adaptive/runner.py

```diff
@@ -26,6 +26,13 @@
 except ModuleNotFoundError:
     with_distributed = False
 
+try:
+    import mpi4py.futures
+    import mpi4py.mpi4py.MPI
```

Comment:
This line is wrong, it should be `from mpi4py import MPI`, or at least `import mpi4py.MPI`.

Reply:
Oops, thanks for checking.